# include direcotry is all caps
# March 15th 2018 by B. Hajgato (Free University Brussels - VUB)
--- mathgl-2.4.1/widgets/image.h.orig	2017-07-20 08:27:56.000000000 +0200
+++ mathgl-2.4.1/widgets/image.h	2018-03-15 23:22:55.211594251 +0100
@@ -18,7 +18,7 @@
 #ifndef IMAGE_H
 #define IMAGE_H
 //-----------------------------------------------------------------------------
-#include <Fl/Fl_RGB_Image.H>
+#include <FL/Fl_RGB_Image.H>
 
 extern MGL_EXPORT Fl_RGB_Image img_load;
 extern MGL_EXPORT Fl_RGB_Image img_save;
--- mathgl-2.4.1/mgllab/mgllab.cpp.orig	2017-07-20 08:27:59.000000000 +0200
+++ mathgl-2.4.1/mgllab/mgllab.cpp	2018-03-15 23:30:39.770144924 +0100
@@ -19,10 +19,10 @@
 #include <locale.h>
 #include <getopt.h>
 #include "mgllab.h"
-#include <Fl/Fl_Spinner.H>
-#include <Fl/Fl_Output.H>
-#include <Fl/Fl_Select_Browser.H>
-#include <Fl/Fl_Printer.H>
+#include <FL/Fl_Spinner.H>
+#include <FL/Fl_Output.H>
+#include <FL/Fl_Select_Browser.H>
+#include <FL/Fl_Printer.H>
 //-----------------------------------------------------------------------------
 #ifndef MGL_DOC_DIR
 #ifdef WIN32
--- mathgl-2.4.1/mgllab/mgllab.h.orig	2017-07-20 08:27:59.000000000 +0200
+++ mathgl-2.4.1/mgllab/mgllab.h	2018-03-15 23:31:31.898973770 +0100
@@ -34,14 +34,14 @@
 #include <FL/Fl_Text_Editor.H>
 #include <FL/Fl_Pixmap.H>
 #include <FL/Fl_Counter.H>
-#include <Fl/Fl_Scroll.H>
+#include <FL/Fl_Scroll.H>
 #include <FL/Fl_Tabs.H>
 #include <FL/Fl_Help_View.H>
-#include <Fl/Fl_Table.H>
-#include <Fl/Fl_Round_Button.H>
-#include <Fl/Fl_Float_Input.H>
-#include <Fl/Fl_Multiline_Input.H>
-#include <Fl/Fl_Multiline_Output.H>
+#include <FL/Fl_Table.H>
+#include <FL/Fl_Round_Button.H>
+#include <FL/Fl_Float_Input.H>
+#include <FL/Fl_Multiline_Input.H>
+#include <FL/Fl_Multiline_Output.H>
 //-----------------------------------------------------------------------------
 #include "mgl2/Fl_MathGL.h"
 //-----------------------------------------------------------------------------
--- mathgl-2.4.1/utils/mglview.cpp.orig	2017-07-20 08:27:56.000000000 +0200
+++ mathgl-2.4.1/utils/mglview.cpp	2018-03-15 23:31:50.030914277 +0100
@@ -23,8 +23,8 @@
 #include "mgl2/mgl.h"
 #if USE_FLTK
 	#include "mgl2/fltk.h"
-	#include <Fl/Fl.H>
-	#include <Fl/Fl_Preferences.H>
+	#include <FL/Fl.H>
+	#include <FL/Fl_Preferences.H>
 #else
 	#include "mgl2/qt.h"
 #endif
